src/bin/menu_bar.rs

```diff
@@ -49,6 +51,9 @@ fn build_ui(application: &gtk::Application) {
     let v_box = gtk::Box::new(gtk::Orientation::Vertical, 10);
 
     let menu = Menu::new();
+    let accel_group = AccelGroup::new();
+    window.add_accel_group(&accel_group);
```

Comment:
Might be useful to explain why you added the accel group on both `window` and `menu`.

Reply:
It seems like `menu.set_accel_group` does two things: it makes the `accel_group` available through `menu.get_accel_group` and it refreshes the `accel_path`s for the menu and its children. Since we don't seem to need either, I decided to get rid of the line.